types/index.d.ts

```diff
@@ -98,6 +98,25 @@ export function render(
   options?: Omit<RenderOptions, 'queries'>,
 ): RenderResult
 
+// TODO JSDOC
+interface RenderHookResult<Result, Props> {
+  rerender: (props?: Props) => void
+  result: {current: Result}
+  unmount: () => void
+}
+
+// TODO JSDOC
+interface RenderHookOptions<Props> {
+  initialProps?: Props
+  wrapper?: React.ComponentType
```

Comment:
just wondering if this should be `React.JSXElementConstructor<{children: React.ReactElement}>` [like in `RenderOptions`](https://github.com/testing-library/react-testing-library/blob/main/types/index.d.ts#L80)?

Reply:
I think you're right. I'll make that change.